modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -76,20 +70,14 @@ $ oc exec -it linuxptp-daemon-5j265 -n openshift-ptp -c cloud-event-proxy -- cur
 
 Returns a list of publishers. If publishers exist, a `200 OK` status code is returned along with the list of publishers.
 
-.Example oc exec curl command
-[source,terminal]
-----
-$ oc exec -it linuxptp-daemon-5j265 -n openshift-ptp -c cloud-event-proxy -- curl --location http://localhost:8080/api/cloudNotifications/v1/publishers
-----
-
-.Example return
+.Example API response
 [source,json]
 ----
 [
  {
   "id": "56e8a064-dc4b-4428-8085-91c18ea07930",
-  "endpointUri": "http://localhost:8080/api/cloudNotifications/v1/dummy",
-  "uriLocation": "http://localhost:8080/api/cloudNotifications/v1/publishers/56e8a064-dc4b-4428-8085-91c18ea07930",
+  "endpointUri": "http://localhost:8089/api/cloudNotifications/v1/dummy",
```

Comment:
instead of dummy endpoint, can you call it either `event` or 'return-event-here'
enpointURI is the return URL for the subscriber running in DU(not sidecar), this will be web services running in DU to receive PTP events

Reply:
"endpointUri": "http://127.0.0.1:9089/event",